way_to_home/place/views.py

```diff
@@ -1 +1,108 @@
 """This module that provides base logic for CRUD of place`s model objects."""
+
+from django.http import HttpResponse, JsonResponse
+from django.views import View
+
+from .models import Place
+
+
+class PlaceView(View):
+    """Class that handle HTTP requests for place model."""
+
+    def post(self, request, place_id=None):
+        """Handle the request to create a new place object."""
+        data = request.body
+        if not data:
+            return HttpResponse('empty json received', status=400)
+
+        data = {
+            'longitude': data.get('longitude'),
+            'latitude': data.get('latitude'),
+            'address': data.get('address'),
+            'name': data.get('name'),
+            'stop_id': data.get('stop_id')
+        }
+
+        # if not place_data_validator(data):
+        #     return HttpResponse('invalid data', status=400)
+
+        place = Place.create(user=request.user, **data)
+        if not place:
+            return HttpResponse('database operation is failed', status=400)
+
+        place = place.to_dict()
+        return JsonResponse(place, status=201)
+
+    def get(self, request, place_id=None):
+        """Handle the request to retrieve a place object or user`s places."""
+        user = request.user
+
+        if not place_id:
+            places = user.places.all()
```

Comment:
Maybe you should use user.places.exclude(name=‘’) because routes create places too, and we do not need to show this places for our user
Also make checking if in places is not empty QuerySet

Reply:
Places that was created from routes has not user_id. So in response will be only user`s places.
If QuerySet empty, it returns an empty list without any exceptions.